src/Shell/PluginShell.php

```diff
@@ -19,6 +19,10 @@
 
 /**
  * Shell for tasks related to plugins.
+ *
+ * @property mixed $Assets
```

Comment:
`@property \Cake\Shell\Task\AssetsTask $Assets`

Similarly the other 2 properties belows are also tasks from `\Cake\Shell\Task\` namespace.